strategies/low_risk/tsconfig.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
could we put this config to the root directory?

Reply:
Typescript is running in /strategies/low_risk only.
We don't need to move that config to root folder.
If we move this to root folder, "includes", "files" options should indicate "/strategies/low_risk". but other projects don't use this config.